### ISSUE TITLE
Training times for SLak t Mask R-CNN on COCO

### ISSUE THREAD

**Issue by JohannesTheo** [VITA-Group/SLaK#20]
Hey there, thanks for the great work!

I'm in the process of re-training slack t Mask R-CNN on COCO so that I can run some additional evaluations for every epoch.

Everything works fine so far but I was wondering if you are aware of any tricks to speed up training and inference?

I've seen [this](https://github.com/DingXiaoH/RepLKNet-pytorch/issues/17) discussion where they mention that `depthwise_conv2d_implicit_gemm` only excels `torch.nn.Conv2d` for large batch size. Since we use small batch sizes in detection, can you confirm this or maybe recall how long the 1x schedule of Mask R-CNN took, roughly?

Would be great if you could share some times so I can verify everything works as expected and I did't introduce a bottleneck 😅

Thanks and best,
Johannes
